.github/workflows/housekeeping.yml

```diff
@@ -0,0 +1,19 @@
+name: Housekeeping
```

Comment:
[nit] I think this has been copied from some other celestiaorg repos but I would prefer renaming to be more descriptive.

```suggestion
name: pr-review-requester
```

In addition, we may consider renaming `housekeeping.yml` to `pr-review-requester.yml`.

## Motivation
The "housekeeping" workflow name communicates very little about what this workflow does. This is especially problematic because the Github actions sidebar retains old workflow names for a while so it can be confusing what some of these do when they don't have descriptive names. 

<img width="453" alt="Screenshot 2024-01-24 at 5 00 56 PM" src="https://github.com/celestiaorg/CIPs/assets/3699047/f6e7e638-6271-454e-97f7-0365b18002f5">

Ref: https://github.com/celestiaorg/celestia-app/blob/b1592c314d4dca77c477e62ee1890970c6ee6e0c/.github/workflows/pr-review-requester.yml#L1

Reply:
SGTM, thanks for the review @rootulp